src/routers/mailingList/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from './mailingList';
```

Comment:
I think you can write all the code in this file. I can't see router's getting too for now

Reply:
All the code in `mailingList.ts` or all the files?
I'm in the habit of only using the index to list exports but in this case it will probably never expand to export anything else but the mailing list route so I see no harm in moving that code to the index

Reply:
Yea just `mailingList.ts`